Finalised Functions/kl_divergence_profile.py

```diff
@@ -6,76 +6,71 @@
 from scipy.spatial import distance_matrix
 from scipy.interpolate import interp1d
 
-def divergence_profile(populations, geometries):
+
+def divergence_profile(populations, geodataframe):
     """
     Description of the function here.
-    
+
     Reference to paper here.
-    
+
     Arguments
     populations : form(s) of input
                   form(s) of input
-    geometries : form(s) of input
+    geodataframe : form(s) of input
                  form(s) of input
     selectioninput1 : form of input
         Description of what this is and what it does here.
     selectioninput2 : form of input
         Description of what this is and what it does here.
-    
+
     Returns
     ----------
     Description of the format of return
-    
+
     Example
     ----------
-    Basic written example here, using test data if possible or creating simple dataframe to run
+    Basic written example here, using test data if possible or
+    creating simple dataframe to run
     """
-    # Creating a distance matrix
-    centroids = geometries.geometry.centroid
+    # Extract populations
+    populations = geodataframe[populations].values.astype(float)
```

Comment:
Sweet,

Thanks so much for going over that, Jeff.

@ljwolf - Jeff and I just ran over the code and refined it further.

We wanted to check your thoughts on simplifying the inputs of the function slightly, such that it is:
* a geodataframe
* a list of the column names to be used within the divergence profile

We'd discussed this collectively before, but it means moving the creation of the population matrix from being a prerequisite to be included within the function (as is seen on lines 35 and 36).

What are your thoughts on that, and do you have any other comments on the current state of the code?